app/src/main/res/layout/activity_orgs.xml

```diff
@@ -10,6 +10,25 @@
         android:layout_height="match_parent"
         tools:context=".ui.organizations.OrganizationProfile">
 
+        <RelativeLayout
```

Comment:
Recycler view for the posts looks good. Nice job on the tab layout functionality